hack/patch.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -xe
+
+kubectl=./cluster-up/kubectl.sh
+
+function wait-for-daemonset() {
+    retries=10
+    while [[ $retries -ge 0 ]]; do
+        sleep 3
+        ready=$($kubectl -n $1 get daemonset $2 -o jsonpath="{.status.numberReady}")
+        required=$($kubectl -n $1 get daemonset $2 -o jsonpath="{.status.desiredNumberScheduled}")
+        if [[ $ready -eq $required ]]; then
+            #echo "Succeeded"
+            break
+        fi
+        ((retries--))
+    done
+}
+
+BAZEL=bazelisk PUSH_TARGETS="virt-controller virt-handler virt-launcher" ./hack/bazel-push-images.sh
```

Comment:
Would it make sense to push `virt-api` here too?

Reply:
sure, should not take too much if it's not changed.

Reply:
I even think that the digest does not change if the code does not change. It should therefore be almost a noop :)